### PR TITLE
Add Readme

### PR DESCRIPTION
**Pull request by arnaudde**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+#### Available parameters
+
+- `Saved model` (DSS saved model): Visual Deep Learning model trained in DSS to convert
+- `Output folder` (DSS managed folder): Folder where the onnx model will be added
+- `Output model path` (String): Path where the onnx model will be stored
+- `Overwrite if exists` (boolean): Whether the model should overwrite the existing file at same path (if it already exists)
+- `Fixed batch size` (boolean): Some runtimes do not support dynamic batch size and thereefore the size should be specified during export.
+- `Batch size` (int) [optional]: Batch size of the model's input
+- `Force input/output to Float (32 bits)` (int): Some runtimes do not support `Double`. Uncheck if your runtime supports `Double`
```
[the remark this answers is not in the document]
Comment (samronsin) [reply README.md:71]: Besides I would phrase it the other way around like "Check if your runtime does not support `Double`" and default to Unchecked.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+
+- `Input folder` (DSS managed folder): Folder where the .h5 model is
+- `Model path`(String): Path to the .h5 model to convert
+- `Output folder` (DSS managed folder) [optional]: Folder where the onnx model will be added. 
+If `Output folder` is left empty the model is added to the input folder.
+- `Output model path` (String): Path where the onnx model will be stored
+- `Overwrite if exists` (boolean): Whether the model should overwrite the existing file at same path (if it already exists)
+- `Fixed batch size` (boolean): Some runtimes do not support dynamic batch size and thereefore the size should be specified during export.
+- `Batch size` (int) [optional]: Batch size of the model's input
+- `Force input/output to Float (32 bits)` (int): Some runtimes do not support `Double`. Uncheck if your runtime supports `Double`
```
[the remark this answers is not in the document]
Comment (samronsin) [reply README.md:100]: Ditto

File: README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+
+![](resource/doc/saved_model_recipe_params.png)
+
+#### Available parameters
+
+- `Output model path` (String): Path where the onnx model will be stored
+- `Overwrite if exists` (boolean): Whether the model should overwrite the existing file at same path (if it already exists)
+- `Fixed batch size` (boolean): Some runtimes do not support dynamic batch size and thereefore the size should be specified during export.
+- `Batch size` (int) [optional]: Batch size of the model's input
+- `Force input/output to Float (32 bits)` (int): Some runtimes do not support `Double`. Uncheck if your runtime supports `Double`
```
[the remark this answers is not in the document]
Comment (samronsin) [reply README.md:139]: Ditto

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (arnaudde) [reply README.md:73]: The thing is that most runtimes do not support `Double` for now

**Comment by samronsin** [comment]
As discussed, LGTM !

**Pull request merged**